openshift-server-mock/src/main/java/io/fabric8/openshift/client/server/mock/OpenShiftServer.java

```diff
@@ -40,6 +41,16 @@ public KubernetesClient getKubernetesClient() {
   }
 
   public NamespacedOpenShiftClient getOpenshiftClient() {
+    if (!new OpenshiftAdapterSupport().isAdaptable(client)) {
+      throw new IllegalArgumentException("Client does not support OpenShift!");
```

Comment:
just pushed a code comment to be more clear why - the issue is the lazy call to the `/apis` REST API (to detect which API groups are supported on the kubernetes/openshift cluster) was causing tests to fail due to the extra REST call appearing the mock server results; hence the eager query